### ISSUE TITLE
Qt imported from PyQt5 by mistake

### ISSUE THREAD

**Issue by chrisemblhh** [nexpy/nexpy#65]
I built nexpy from the github master on Fedora 22 using
```
virtualenv --system-site-packages venvs/nexpy
source venvs/nexpy/bin/activate
pip install --upgrade pip
pip install --editable {path to github checkout}
```

Running nexpy gives
```
$ nexpy
Traceback (most recent call last):
  File "/home/chris/.venvs/nexpy2/bin/nexpy", line 9, in <module>
    load_entry_point('NeXpy==0.6.5', 'gui_scripts', 'nexpy')()
  File "/home/chris/projects/python/nexpy/src/nexpy/nexpygui.py", line 20, in main
    from nexpy.gui.consoleapp import main
  File "/home/chris/projects/python/nexpy/src/nexpy/gui/consoleapp.py", line 30, in <module>
    from mainwindow import MainWindow
  File "/home/chris/projects/python/nexpy/src/nexpy/gui/mainwindow.py", line 44, in <module>
    from treeview import NXTreeView
  File "/home/chris/projects/python/nexpy/src/nexpy/gui/treeview.py", line 226, in <module>
    class NXSortModel(QtGui.QSortFilterProxyModel):
AttributeError: 'module' object has no attribute 'QSortFilterProxyModel'
```

**Comment by rayosborn** [comment]
I apologize for taking a while to get to this. In principle, NeXpy should have been compatible with PyQt5, since we use the qtconsole.qt function, which creates a PyQt4-compatible set of imports. This mostly involves functions moving from QtGui to QtWidgets, but it missed the fact that some modules, including QSortFilterProxyModel had been moved from QtGui to QtCore! I could fix that, but It turns out that creates other issues, at least on a Mac. So I have now switched to using the matplotlib qt_compat module to base imports, so that it honors the matplotlib.use call. I will be issuing a pull request from my own repository soon.
